src/Cryptol/Parser.y

```diff
@@ -69,6 +69,7 @@ import Paths_cryptol
   'type'      { Located $$ (Token (KW KW_type   ) _)}
   'newtype'   { Located $$ (Token (KW KW_newtype) _)}
   'enum'      { Located $$ (Token (KW KW_enum) _)}
+  'deriving'  { Located $$ (Token (KW KW_deriving) _)}
```

Comment:
Does adding `deriving` to `ident` cause conflicts?  If not, we should probably add it there.

Reply:
I see this when I build the parser with `'deriving'` in `ident`:
```
shift/reduce conflicts:  2
reduce/reduce conflicts: 2
```
so I guess the answer is yes?